test/e2e/create_test.go

```diff
@@ -133,6 +133,53 @@ func (tc *testContext) testBYOHConfiguration(t *testing.T) {
 		t.Skip("BYOH testing disabled")
 	}
 
+	err := tc.configureBYOHWithMachineSets()
+	require.NoError(t, err, "error configuring BYOH with MachineSets")
+
+	t.Run("VM is configured by ConfigMap controller", func(t *testing.T) {
+		err := tc.waitForWindowsNodes(gc.numberOfBYOHNodes, false, false, true)
+		assert.NoError(t, err, "Windows node creation failed")
+	})
+}
+
+// configureBYOHWithMachineSets configures BYOH instances using MachineSets
+func (tc *testContext) configureBYOHWithMachineSets() error {
```

Comment:
maybe rename this to something like `newBYOHConfigMapFromMachineSet`?

Reply:
Not sure about that, do you anticipate `newBYOHConfigMapFromMachineSet` to return a reference of the `windows-instances` ConfigMap?

Reply:
No, I'm just thinking that `configureBYOHWithMachineSets` doesn't really capture what the function is doing. The configuration of BYOH instances will be done by WMCO, this just enables it.

Alternatively maybe `createBYOHConfigMapFromMachineSet`?

Reply:
Makes sense. Updated to `provisionBYOHConfigMapWithMachineSet`  in https://github.com/openshift/windows-machine-config-operator/pull/865/commits/3f4f80027d114efe5e141bed1dcc3c812560751e